.github/workflows/darwin.yml

```diff
@@ -1,60 +0,0 @@
-name: "darwin"
-
-on:
-  push:
-    branches: [master]
-  workflow_dispatch:
-
-concurrency:
-  group: "darwin"
-  cancel-in-progress: false
-
-jobs:
-  deploy:
-    environment:
-      name: darwin
-    strategy:
-      fail-fast: false
-      matrix:
-        host: [darwin01, darwin02]
-    runs-on: ubuntu-latest
-    timeout-minutes: 15
-    steps:
-      - uses: actions/checkout@v4
-      - name: Install Nix
-        uses: cachix/install-nix-action@v30
-        with:
-          extra_nix_config: |
-            accept-flake-config = true
-            experimental-features = nix-command flakes
-      - name: Setup SSH
-        env:
-          SSH_KEY: "${{ secrets.SSH_KEY }}"
-        run: |
-          mkdir -p "$HOME/.ssh"
-          cat >>"$HOME/.ssh/id_ed25519" <<EOF
-          $SSH_KEY
-          EOF
-          chmod 600 "$HOME/.ssh/id_ed25519"
-          known_hosts="$(nix eval .#nixosConfigurations.build01.config.environment.etc.'ssh/ssh_known_hosts'.text)"
-          echo -e "$known_hosts" >>"$HOME/.ssh/known_hosts"
-          cat >>"$HOME/.ssh/config" <<EOF
-          Host *
-            IdentityFile ~/.ssh/id_ed25519
-            IdentitiesOnly yes
-          EOF
-      - name: Deploy
-        run: |
-          ./inv deploy --hosts ${{ matrix.host }}
-      - name: Alert on failure
```

Comment:
We won't have this alert with effects but if it fails the buildbot status check will be red on master and darwin deploy failures were mostly a just problem on the m1s from hetzner, it has only failed a couple of times since we switched to the better m2/m4 machines.